SEM_2_PRACTICALS/prac_06/guitars.py

```diff
@@ -0,0 +1,49 @@
+from SEM_2_PRACTICALS.prac_06.guitar import Guitar
+
+MIN_YEAR = 0
+MIN_COST = 0
+
+
+def main():
+    guitars = []
+    print("My guitars!")
+    get_guitars(guitars)
```

Comment:
This is not how we write "get" functions. Look at any of our examples. Why pass in an empty list to populate instead of returning a new list?